extensions/Voting/Extension.pm

```diff
@@ -899,7 +899,8 @@ sub _remove_votes {
 sub _confirm_if_vote_confirmed {
   my ($id, $product, $timestamp) = @_;
   my $bug = ref $id ? $id : new Bugzilla::Bug({id => $id, cache => 1});
-  $product ||= $bug->product_obj;
+  $product   ||= $bug->product_obj;
+  $timestamp ||= Bugzilla->dbh->selectrow_array('SELECT LOCALTIMESTAMP(0)');
```

Comment:
I can't find where `$timestamp` is used 🤔

Reply:
https://github.com/mozilla-bteam/bmo/blob/master/extensions/Voting/Extension.pm#L911

It happens when a bug is updated and triggers a voting change, we want the voting related comment to have the same timestamp as the other bug change(s).